app/scripts/modules/core/src/pipeline/config/stages/executionWindows/executionWindows.controller.js

```diff
@@ -145,6 +147,9 @@ module.exports = angular.module('spinnaker.core.pipeline.stage.executionWindows.
       });
     });
 
+    this.enableSkipWindowText = !!$scope.stage.skipWindowText;
```

Comment:
i'm just gonna throw out there that — while more verbose — this might be a little clearer as `enableCustomSkipWindowText`

Reply:
![ok](https://bukk.it/okwiththis.png)